### ISSUE TITLE
Don't show the Download Apps button.

### ISSUE THREAD

**Issue by MagmaSlime123** [smartfrigde/armcord#496]
**Is your feature request related to a problem? Please describe.**
Title.

![image](https://github.com/ArmCord/ArmCord/assets/51277766/71619b5d-92cf-474f-a6e3-a1de4e82a639)

**Describe the solution you'd like**
Don't show the Download Apps button.

**Describe alternatives you've considered**
N/A?

**Additional context**
N/A.

**Comment by smartfrigde** [comment]
Fixed in 3.2.6

**Comment by Davilarek** [comment]
it's still there for me...
![obraz](https://github.com/ArmCord/ArmCord/assets/62715937/29a19247-85fd-4e01-96a1-98c9c1463555)

**Comment by whiteawake** [comment]
It's showing for me also.

**Comment by mucookul** [comment]
I still have it

**Comment by u254254** [comment]
I still have it too
<img width="192" alt="ArmCord_06wqIpNmfk" src="https://github.com/ArmCord/ArmCord/assets/72965213/f524068e-296a-4ef0-a1ce-061a9ade44a9">

**Comment by mucookul** [comment]
I got a fix, if you are using Vencord, put this in your quickcss: 

```
.listItem_fa7b36:nth-child(6),
.listItem_fa7b36:nth-child(7) {
  display:none;
}
```

**Comment by SomeAspy** [comment]
This is still present in 3.2.8, however without an icon.
![image](https://github.com/user-attachments/assets/b9efec28-db35-47f1-a284-c7957442912b)

**Comment by smartfrigde** [comment]
Fixed in 1.0.0
